object/check_password_expired.go

```diff
@@ -0,0 +1,43 @@
+package object
+
+import (
+	"time"
+
+	"github.com/casvisor/casvisor-go-sdk/casvisorsdk"
+)
+
+func CheckPasswordExpired(user *User) bool {
+	organization, err := GetOrganizationByUser(user)
+	if err != nil {
+		return false
+	}
+	passwordExpireDays := organization.PasswordExpireDays
+	if passwordExpireDays <= 0 {
+		return false
+	}
+	lastChangeTime, err := getLastTimeOfAction(organization.Name, user.Name, "set-password")
```

Comment:
Add a field to user to record the last password change time, don't iterate on logs